### ISSUE TITLE
can not print web page or email 

### ISSUE THREAD

**Issue by casperzimm** [brave/browser-laptop#1246]
I can print from my other browsers just fine but can not from eamil or a web page. What am I missing

**Comment by weems** [comment]
Can you provide a specific page you try to print from which fails?

**Comment by casperzimm** [comment]
http://www.clockworks.com/clock-repair/clock-repair-help-2.html#c

I went to this web page and it prints when I use IE. In brave i used
control p and the menu to try and print it and it did not print. the print
Icon comes up in both cases but it does not print. same thing happens in
google mail.

I am using win 10 64 bit enterprise.

Brave 0.8.3 eletron 0.37.3 LIBC.... 49.9.385.33 Node 6.0.0.0-pre update
channel DEV

On a side note spellcheck does not work either.
Love this browser alot. thanks for making this happen.

if there is anything else I can do let me know.

**Comment by weems** [comment]
@bbondy I do not have a printer on me at the moment, but I was able to Print to PDF via Preview on OSX 10.11.4 and Brave 0.8.3.

**Comment by casperzimm** [comment]
I was unable to print to a pdf.

**Comment by casperzimm** [comment]
running brave as administrator had no effect.

**Comment by casperzimm** [comment]
​I brought up the printer from the brave menu, I went to the printer
properties and I copuld print a test page.​

**Comment by ParrotSlave** [comment]
I haven't tried a physical printer (I didn't want to waste the paper), but I could not get Brave to print either to Nuance's virtual printer (the Scansoft virtual driver in this case) or the Microsoft XPS Document Writer. In both cases, the print dialog comes up, and I point the virtual printer to my intended save location, and click print: the dialog box disappears, but nothing shows up in the folder. I tried several folders on different drives, thinking that it might be a permission issue on the original location. Then I realized that if the document were really there, trying to print it again to the same folder would give an error message about a file of that name already in the folder if I kept the same name, and the dialog box did not show a previous file. Then I had the inspiration to select "View Resulting PDF", but nothing happened. So apparently the documents are in an alternate reality. I tried this on several different web pages, including this very one, with the same result. This is on Win8.1 64-bit, not from the administrator account, but  from an account with admin credentials.
